build_scripts/windows_bash_nuttx.sh

```diff
@@ -35,9 +35,10 @@ fi
 
 # Common Dependencies
 echo "Installing common dependencies"
-sudo add-apt-repository ppa:george-edison55/cmake-3.x -y
 sudo apt-get update
-sudo apt-get install python-argparse git git-core wget zip python-empy python-toml python-numpy cmake build-essential genromfs vim-common -y
+sudo apt-get install python-argparse git git-core wget zip python-empy python-toml python-numpy build-essential genromfs vim-common software-properties-common -y
```

Comment:
So just FMI, why can't we use the cmake from the normal repo? This is why we need software-properties-common here right?

Reply:
Actually I see that it is because we need a different cmake. But again - "why"? The ubuntu.sh script seems to install cmake fine.

I'd quite like to dump these scripts and move to ubuntu.sh, which is why I'm asking if perhaps we can avoid this change.

Reply:
Agreed. We need to check what cmake version we require.

According to https://github.com/PX4/Firmware/blob/60e5e0821a0ee64c42cd77e85e0c0c39526dca88/CMakeLists.txt#L102 we need 3.2.
Ubuntu 16.04 comes with 3.5.1, so that should be fine.

Reply:
So I'm a little lost, but does that mean most of the changes except LSB-core addition should be reverted?

Reply:
It means that we should probably remove the ppa for cmake because it's not required anymore, and that means we also would not need the `software-properties-common` anymore.

Reply:
Great. I'll leave this fix to you when you do the thingy for https://github.com/PX4/Firmware/pull/12827#issuecomment-527953817

I'll add that to ubuntu.py